scopesim/effects/spectral_trace_list_utils.py

```diff
@@ -957,44 +939,44 @@ def get_affine_parameters(coords):
     return rotations, shears
 
 
-def sanitize_table(tbl, invalid_value, wave_colname, x_colname, y_colname,
-                   spline_order=4, ext_id=None):
-
-    y_colnames = [col for col in tbl.colnames if y_colname in col]
-    x_colnames = [col.replace(y_colname, x_colname) for col in y_colnames]
-
-    for x_col, y_col in zip(x_colnames, y_colnames):
-        wave = tbl[wave_colname].data
-        x = tbl[x_col].data
-        y = tbl[y_col].data
-
-        valid = (x != invalid_value) * (y != invalid_value)
-        invalid = np.invert(valid)
-        if sum(invalid) == 0:
-            continue
-
-        if sum(valid) == 0:
-            logging.warning("--- Extension {} ---"
-                            "All points in {} or {} were invalid. \n"
-                            "THESE COLUMNS HAVE BEEN REMOVED FROM THE TABLE \n"
-                            "invalid_value = {} \n"
-                            "wave = {} \nx = {} \ny = {}"
-                            "".format(ext_id, x_col, y_col, invalid_value,
-                                      wave, x, y))
-            tbl.remove_columns([x_col, y_col])
-            continue
-
-        k = spline_order
-        if wave[-1] > wave[0]:
-            xnew = InterpolatedUnivariateSpline(wave[valid], x[valid], k=k)
-            ynew = InterpolatedUnivariateSpline(wave[valid], y[valid], k=k)
-        else:
-            xnew = InterpolatedUnivariateSpline(wave[valid][::-1],
-                                                x[valid][::-1], k=k)
-            ynew = InterpolatedUnivariateSpline(wave[valid][::-1],
-                                                y[valid][::-1], k=k)
-
-        tbl[x_col][invalid] = xnew(wave[invalid])
-        tbl[y_col][invalid] = ynew(wave[invalid])
-
-    return tbl
+# def sanitize_table(tbl, invalid_value, wave_colname, x_colname, y_colname,
```

Comment:
Do you want to keep that? It could be retrieved from previous commits, although that can be a hassle.